oaknational/Oak-Web-Application
# [CUR-456] fix: added aria-live="polite" to "units highlighted" text

## Description

Music year: 1996

- Added `aria-live="polite"` to `"units highlighted"` text

## Issue(s)
Fixes `CUR-456`

## How to test

When a user selects a unit, the number of highlighted units is displayed in text, and when a user filters through the years the results update, but this information isn’t conveyed automatically to screen reader users.

1. Go to `/teachers/curriculum`
2. Select a subject and school phase
3. Filter by the thread (highlight a thread)
   - The number of units highlighted is announced to screen reader
4. Select a year group
   -  The number of units highlighted should be re-announced

## Checklist
